components/browser/session-storage/src/main/java/mozilla/components/browser/session/storage/RecoverableBrowserState.kt

```diff
@@ -11,8 +11,10 @@ import mozilla.components.browser.state.state.recover.RecoverableTab
  *
  * @param tabs The list of restored tabs.
  * @param selectedTabId The ID of the selected tab in [tabs]. Or `null` if no selection was restored.
+ * @param inProgressMediaTabId The ID of the last tab known to have played media.
  */
 data class RecoverableBrowserState(
     val tabs: List<RecoverableTab>,
-    val selectedTabId: String?
+    val selectedTabId: String?,
+    val inProgressMediaTabId: String?
```

Comment:
Why does this need to be restored? On restore this is not in progress - either it again starts playing (and then becomes in progress) or not?

Reply:
If the app was closed in the background, when the user opens it again we'd show the last tab which had media playing allowing him to easily resume. That's how I understood the feature.